main_routers/config_router.py

```
@@ -93,11 +93,25 @@ async def get_character_reserved_fields():
 
 def _get_live3d_sub_type(catgirl_config: dict) -> str:
     """判断 Live3D 模式下应使用 VRM 还是 MMD 渲染器。
-    优先检查 MMD 路径（因为 VRM 是旧字段，可能遗留非空值）。"""
+    优先使用持久化的子类型；缺失或失效时再按模型路径回退判断。"""
```

Ruff (0.15.9)

[warning] Docstring contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF002)